cypress/page-objects/homePage.js

```diff
@@ -0,0 +1,31 @@
+class Header {
+  getHomeCTA() {
+    return cy.get('.sc-kstrdz a[href="/"]');
+  }
```

Comment:
we need to hop on a call to explain how we are going to use class methods and getters for composing page objects